Blish HUD/Controls/_Types/IWindow.cs

```diff
@@ -24,5 +29,16 @@ public interface IWindow {
         /// </summary>
         bool CanClose { get; }
```

Comment:
We might want to incorporate `CanClose` into this somehow so that a window that implements `IWindow` that doesn't intend on being closed by user input (so no X or no closing on ESC) can exist.

Might just be that if the active window is a window that has `CanClose` set to false, we just ignore it and let the ESC pass through to the game.  Thoughts?

Reply:
I have added a check to conditionally handle the escape key depending on the CanClose property of the active window.